orchestrators/ProcurementOperations/src/Coordinators/VendorRecommendationCoordinator.php

```
@@ -0,0 +1,92 @@
+
+            if ($candidateEnrichment === []) {
+                $candidates[] = $candidate;
+                continue;
+            }
+
+            $candidates[] = $this->applyEnrichment($candidate, $candidateEnrichment);
+        }
+
+        usort(
+            $candidates,
+            static fn (VendorRecommendationScoredCandidate $a, VendorRecommendationScoredCandidate $b): int =>
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $a. Configured minimum length is 3. (undefined)

(ShortVariable)